server/app/repository/CategoryRepository.java

```diff
@@ -0,0 +1,60 @@
+package repository;
+
+import com.google.common.collect.ImmutableList;
+import io.ebean.DB;
+import io.ebean.Database;
+import javax.inject.Inject;
+import models.CategoryModel;
+
```

Comment:
**suggestion:** Let's also add a javadoc on the class for consistency with other repository files.

Reply:
Done